apiserver/facades/controller/firewaller/firewaller.go

```diff
@@ -323,20 +333,23 @@ func (f *FirewallerAPI) GetAssignedMachine(ctx context.Context, args params.Enti
 	return result, nil
 }
 
+// NOTE(jack-w-shaw): temporarily dropped pending application endpoint DQLite
+// implementation
```

Comment:
I am much more of a fan of creating a task on the application epic, which links to the 3.6 code, which states that it should be reinstated.